### DISCUSSION TITLE
Error installing Pinia in a Nuxt 3 project

### DISCUSSION THREAD

**Discussion by taikio** [vuejs/pinia#1897] [Help and Questions] [answered]
### Reproduction

https://github.com/vuejs/pinia

### Steps to reproduce the bug

1 - Create a new nuxt 3 project
2 - Run `npm install` to install default nuxt dependencies
3 - Run `npm install @pinia/nuxt`
4 - Add `@pinia/nuxt` to modules array on `nuxt.config.ts`
5 - Run `npm run dev`

### Expected behavior

the application should run normally

### Actual behavior

Following the steps to reproduce will rise the error **Cannot start nuxt:  Cannot find module 'pinia/dist/pinia.mjs'**

### Additional information

i am using node version 16.13.2

**Comment by posva** [comment, marked as answer]
See https://github.com/piniajs/example-nuxt-3 for an example. You need to install `pinia` as well

**Reply by lmc3s3f0r** [reply]
Shouldn't that be mentioned in the docs? https://pinia.vuejs.org/ssr/nuxt.html

The instructions there don't work.

**Reply by posva** [reply]
It has changed in the meantime. The module now installs pinia too
